server/handlers/tagHandler.js

```diff
@@ -8,13 +8,13 @@ const {
 const tagGetQuerySchema = Joi.object({
   limit: Joi.number().integer().greater(0).less(101),
   offset: Joi.number().integer().greater(-1),
-  owner_id: Joi.string(),
+  owner_id: Joi.string().default(null),
 }).unknown(false);
 
 const tagPostQuerySchema = Joi.object({
   isPublic: Joi.boolean().required(),
   name: Joi.string().required(),
-  owner_id: Joi.string().required(),
+  owner_id: Joi.string().allow(null).default(null),
```

Comment:
For the default to work, we would have to get the value returned by the validateAsync function and then pass that to the create tag function instead of req.body